src/ripple/protocol/impl/tokens.cpp

```diff
@@ -30,13 +30,22 @@
 
 namespace ripple {
 
-static char rippleAlphabet[] =
+static constexpr char const* const xrplAlphabetForward =
```

Comment:
```suggestion
static constexpr char const* const rippleAlphabetForward =
```

Reply:
Also in other places...

Reply:
Can you explain what you mean here? I am not understanding. You think it should use ripple? But the name 'ripple' is belong to the company and not the software.

Reply:
The software is called `rippled`, the company is called "Ripple Labs" and it just holds the copyright for the name "Ripple" (the project named "Ripple" is much older than that). Heck, even that folder is `src/ripple/...`. Previous code used `rippleAlphabet`, changing it to `xrplAlphabet` just puts the ISO-4217 shorthand of a cryptocurrency in the name for no good reason.

Reply:
Aha
Yes I understand you now and I prefer the name ripple also but the company is using that now and so to continue use of ripple seems causes a lot of confusion.
Maybe the company can answer on what this should be because the name get used again elsewhere.

Reply:
The company is actually still called "Ripple Labs Inc", it just uses the name "Ripple" in its branding. See for example https://ripple.com/terms-of-use/ ("These Terms of Use (“Terms”) apply to your access to, and use of, the websites of **Ripple Labs Inc.** and its subsidiaries and affiliated companies (“Ripple,” “we” or “us”), including www.ripple.com (collectively, the “Site”)."), in their privacy policy you also see the full company name (something you should maybe be aware of, if you are impersonating their CEO... ;-) ).

I guess someone working there will eventually review and resolve this.

Reply:
I think we all like the name 'ripple' for this wonderful, collaborative project and feel an attachment to it. And if it were up to me I'd have kept it for the project and this could stay `rippleAlphabet`. But, feelings aside, there's reality. And, as has been pointed out, the naming can cause confusion.

So, since I work "there", let me play Solomon and split the proverbial baby so that we can all move forward: how about simply calling these variables `alphabet` and `reverseAlphabet` and avoiding this whole situation?

Rationale: your car doesn't use "FordNuts" or "FordBolts". It just uses "nuts" and "bolts".

Reply:
I changed variable names to 'alphabetForward' and 'alphabetReverse' as you have asked

Maybe Ripple can see the problem the name 'Xrp Ledger' has and can allow the name 'ripple' to be used again in future?